README.md

```diff
@@ -183,6 +183,82 @@ Blank nodes are not supported.
   }
 }
 ```
+#### Resource Object Fragmentation Shape
+
+A fragmentation strategy that groups triples by (subject) resources.
+It generates shapes and shape index files in the root of the iri container for each information type defined in `shapeDirectory`. 
+The `shapeDirectory` must contain a `config.json` following the exemple below.
+The `shape` must be a path relative to the `shapeDirectory` and the `directory` must be the name of container of the subject.
+The shape located at `shape` must be of the format [`ShExC`](https://shex.io/shex-semantics/index.html#shexc).
+
+`./shape/config.json`
```

Comment:
Is there a reason we can't just include these contents as parameters to `FragmentationStrategyShape`?
That would save us one level of indirection.